spec/requests/admin/invoices_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Admin::InvoicesController, type: [:request, :admin] do
```

Comment:
RSpec/FilePath: Spec path should end with `admin/invoices_controller*_spec.rb`. (https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/FilePath)